### ISSUE TITLE
Error in SELinux detection

### ISSUE THREAD

**Issue by ljlamarche** [EarthCubeInGeo/resen#7]
After installing resen v2019.1.0rc1, I can no longer open the resen tool.

```
$ resen
```
Throws the following error:
```
Traceback (most recent call last):
  File "/anaconda2/envs/python3.6/bin/resen", line 10, in <module>
    sys.exit(main())
  File "/anaconda2/envs/python3.6/lib/python3.6/site-packages/resen/resencmd.py", line 288, in main
    res = resen.Resen()
  File "/anaconda2/envs/python3.6/lib/python3.6/site-packages/resen/Resen.py", line 47, in __init__
    self.bucket_manager = BucketManager(self.base_config_dir)
  File "/anaconda2/envs/python3.6/lib/python3.6/site-packages/resen/Resen.py", line 138, in __init__
    self.selinux = self.__detect_selinux()
  File "/anaconda2/envs/python3.6/lib/python3.6/site-packages/resen/Resen.py", line 583, in __detect_selinux
    p = Popen(['/usr/sbin/getenforce'], stdin=PIPE, stdout=PIPE, stderr=PIPE)
  File "/anaconda2/envs/python3.6/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/anaconda2/envs/python3.6/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/usr/sbin/getenforce': '/usr/sbin/getenforce'
```
It looks like the problem is in the `__detect_selinux` module in the `BucketManager` class, where it tried to open a pipe to `/usr/sbin/getenforce`.  This doesn't exist on my system (Mac OSX).  It looks like I have a `/usr/sbin/` directory, but it doesn't contain `getenforce`.

**Comment by pmreyes2** [comment]
We could test if it is a Linux system before detecting if it is SElinux:
>>> import os
>>> os.uname()[0]
The output could be "Linux" or "Darwin" (mac)

I see that this fact is used later when setting permission for storage. If it is SElinux, the parameter "Z" is added to permissions. The Z parameter might be needed in mac too, I need to test that. I remember I was giving Z permissions when adding storage to a docker container in my mac, but I don't know if that is needed or not. I'll test that.

**Comment by asreimer** [comment]
@pmreyes2 no need. And MacOS does not need z or Z: https://docs.docker.com/storage/bind-mounts/

The solution is to wrap things up in a

    try:
        the code that currently exists
    except FileNotFoundError:
        return False

**Comment by asreimer** [comment]
Fixed by #8
